mlflow/server/js/src/model-registry/components/SchemaTable.test.tsx

```diff
@@ -6,14 +6,26 @@
  */
 
 import React from 'react';
-import { shallow, render } from 'enzyme';
 import { SchemaTable } from './SchemaTable';
-import { Table } from 'antd';
 import { MemoryRouter } from '../../common/utils/RoutingUtils';
-import { mountWithIntl } from '../../common/utils/TestUtils';
+import { fireEvent, renderWithIntl } from '../../common/utils/TestUtils';
+
+function clickHeaderRow(container: HTMLElement, rowIndex: number): void {
+  // click to render inputs table
+  const rows = container.querySelectorAll('tr.section-header-row');
+  if (rows.length < rowIndex + 1) {
+    throw new Error("Couldn't find the row to click");
+  }
+  fireEvent(
+    rows[rowIndex],
+    new MouseEvent('click', {
+      bubbles: true,
+      cancelable: true,
+    }),
+  );
+}
```

Comment:
Did you try using
```
import userEvent from '@testing-library/user-event';
...
userEvent.click(rows[rowIndex]);
```
? 
It's a bit higher level function that should be automatically covering more edge cases (e.g. with bubbling etc) and be more convenient

Reply:
done! thanks for the pointer